addons/mail/static/src/core/common/persona_model.js

```diff
@@ -38,6 +38,9 @@ export class Persona {
     }
 
     get emailWithoutDomain() {
+        if (!this.email) {
+            return "";
+        }
```

Comment:
I don't think this is the correct place to fix it, it looks like we should pre-filter records on which this getter is called instead to exclude them, or if we want to still display them, at least have a value instead of empty.

Reply:
Changes done! (we show partner name if there's no email)